jobs/seed_repos.groovy

```diff
@@ -0,0 +1,53 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+name = 'deis-seed-repos'
+repoName = 'seed-repo'
+
+job(name) {
+  description """
+    <p>Runs deis/seed-repo against all of the workflow components.</p>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/${repoName}")
+      }
+      branch('master')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    githubPush()
+  }
+
+  wrappers {
+    timeout {
```

Comment:
we probably don't need a `timeout` closure for this job as it _should_ be pretty quick...